MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/Power-cycle-MTM2-cabinet.rst

```diff
@@ -0,0 +1,174 @@
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+Power Cycle MTM2 cabinet
+#########################
+
+
+
+.. _Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+This procedure mirrors the `Power-Cycle section under M2 handling errors`_ authored by Te-Wei, albeit presented in the format of Observatory Operations Documentation.
+
+.. _`Power-Cycle section under M2 handling errors`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#power-cycle
+
+This procedure is the final option for resolving an MTM2 issue during the night. It's utilized when the M2 GUI consistently displays alarms, and the MTM2 cannot transition to a closed-loop status following the steps outlined in the MTM2 Recovery procedure.
+
+.. need to add link to the corresponding and not yet in the site "MTM2 recovery procedure" page
+.. test that works  
+.. :doc:`MTM2 Recovery </MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/MTM2-Non-standard-Procedures-index>`
+.. The correct webpage would be in M2 Troubleshooting so </MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/M2-Troubleshooting/MTM2-Recovery>`
+
+.. _Precondition:
+
+Precondition
+============
+
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+.. note::
+    Ensure you have access to 1password and are a member of the Operators vault. If not, request access for the 1password Vault Operators via an `IT Jira ticket`_.
+
+.. _`IT Jira ticket`: https://rubinobs.atlassian.net/jira/
+
+- Consult with the hardware or system engineers to obtain approval before proceeding with this procedure.
+
+- Ensure you are connected to the control room network as VPN cannot reach it.
+
+- You should only do this procedure if:
+
+  - Following MTM2 Recovery procedure doesn't fix the issue, this means re-starting the control system fails to clear alarms and you are unable to set MTM2 to close-loop.
+
+.. add link to MTM2 Recovery 
+
+  - You cannot ping or ssh the M2 controller  m2-crio-controller01.cp.lsst.org (on TMA) **m2-crio-controller02.cp.lsst.org** (on level 3).
```

Comment:
Is this supposed to be commented out?  If not, hostnames are represented in italics between one asterisk.